docs-markdown/package.json

```diff
@@ -474,4 +474,4 @@
     "usePathForSuiteName": "true"
   },
   "license": "MIT"
-}
\ No newline at end of file
+}
```

Comment:
I dont see any packages that got updated?

Reply:
Hi @bharney0 , the changes can be found in the package-lock.json file/diff.  Here's an example:
https://github.com/microsoft/vscode-docs-authoring/pull/340/files#diff-5fd8a3ade18623e5edbcb3bb4b1bc36dR2553

Thanks